### PR TITLE
switch: use audren for audio

### PR DESCRIPTION
**Pull request by Cpasjuste**

### CODE REVIEW

**Comment by Cpasjuste** [comment]
Do not merge yet, needs more testing on various homebrews. Pull request made for review.

File: src/audio/switch/SDL_switchaudio.c [hunk 1]
```diff
@@ -33,35 +33,51 @@
...
+    res = audrvCreate(&this->hidden->driver, &arConfig, 2);
+    if (R_FAILED(res)) {
+        return SDL_SetError("audrvCreate failed (0x%x)", res);
+    }
+    this->hidden->audr_driver = true;
+
     test_format = SDL_FirstAudioFormat(this->spec.format);
     while ((!supported_format) && (test_format)) {
-        if (test_format == AUDIO_S16LSB) {
+        if (test_format == AUDIO_S8
```
Comment (fincs) [inline src/audio/switch/SDL_switchaudio.c]: audren only supports PCM16 and DSPADPCM.

Comment (Cpasjuste) [reply src/audio/switch/SDL_switchaudio.c]: fixed in 88b2394eb3938188011af064aec69f002e5bd3f6

**Comment by rsn8887** [comment]
I can confirm this works perfectly in UAE.

**Comment by rsn8887** [comment]
It also works perfectly with Rocks'n'Diamonds and Mirrormagic.

**Comment by Cpasjuste** [comment]
:) (also works fine in my emu's)

**Pull request merged**